packages/forklift-console-plugin/src/plans/create/CreatePlanWizard.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { useForkliftTranslation } from 'src/utils';
+
+import { Form, Title, Wizard, WizardStep, WizardStepType } from '@patternfly/react-core';
+
+import { GeneralInformationForm } from './steps/GeneralInformationForm';
+import { defaultCurrentStep, planStepIndexes, planStepNames, PlanWizardStepId } from './constants';
+import { CreatePlanWizardFooter } from './CreatePlanWizardFooter';
+
+import './CreatePlanWizard.style.scss';
+
+export const CreatePlanWizard: FC = () => {
+  const { t } = useForkliftTranslation();
+  const form = useForm({ mode: 'onChange' });
+  const [currentStep, setCurrentStep] = useState<WizardStepType>(defaultCurrentStep);
+  const { formState, getValues } = form;
+  const formValues = getValues();
+
+  const onSubmit = () => console.log('SUBMITTED: ', formValues);
+
+  const getStepProps = (id: PlanWizardStepId) => ({
+    id,
+    name: planStepNames[id],
+    isDisabled:
+      currentStep?.index < planStepIndexes[id] && Object.keys(formState?.errors).length > 0,
```

Comment:
could you please add in a shared place this function of isEmpty? https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/utils/utils/utils.ts#L22

We will use it widely instead of checking length or if there are values